src/Console/Commands/JsonApiGeneratorCommand.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+namespace CloudCreativity\LaravelJsonApi\Console\Commands;
+
+use Illuminate\Console\GeneratorCommand as LaravelGeneratorCommand;
+use Illuminate\Filesystem\Filesystem;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+
+abstract class JsonApiGeneratorCommand extends LaravelGeneratorCommand
```

Comment:
Sorry for not spotting this before, but the convention in the package would be to call this class `AbstractGeneratorCommand`.